### DISCUSSION TITLE
For some reason my sensor.solax_today_s_solar_energy reverts to zero at 2300 hours on stats

### DISCUSSION THREAD

**Discussion by UK-JustinSmall** [BottlecapDave/HomeAssistant-OctopusEnergy#739] [Q&A]
For some reason my sensor.solax_today_s_solar_energy reverts to zero at 2300 hours on stats resulting in all of January showing as no solar being produced, which is not correct. 

I suspect it just reports the hour between 2300 and 0000 which is obviously going to be zero. Not sure what changed but hoping someone can point me in the right direction to fix this.

As you will see it affects report charts for daily production in home assistant

![solax 1 hour error](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/129121587/ab3604c9-162d-43ac-8828-e24fa942d5ff)

**Comment by BottlecapDave** [comment]
As this is not related to the integration, you might be better asking themis question in the main HA forum?
